frontend/.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:prettier/recommended"
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+        "react/react-in-jsx-scope": "off",
```

Comment:
Why?

Reply:
this rule suppress errors for missing 'import React' in files

Reply:
True, is this missing React import an issue during builds to raw JavaScript?

Reply:
Not really, at last meeting we was building an app with `npm run build` and it worked despite not importing React in components. 
Other sources: [stackoverflow](https://stackoverflow.com/questions/71725865/do-i-need-to-import-react-in-every-file) [dev.to](https://dev.to/titungdup/you-no-longer-need-to-import-react-from-react-3pbj)